devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy

```diff
@@ -96,7 +96,7 @@ def parse(String description) {
 private void createChildDevices() {
 	def i = 2
 	addChildDevice("Child Switch Health", "${device.deviceNetworkId}:0${i}", device.hubId,
```

Comment:
this DNI won't work with the `findChildDevice` call elsewhere in the DTH because of the `0`

Reply:
It works because the endpoint in the message includes the leading 0